server/data/assessForEarlyReleaseApiClient.ts

```diff
@@ -15,29 +22,36 @@ export default class AssessForEarlyReleaseApiClient {
   }
 
   async getCaseAdminCaseload(prisonCode: string): Promise<OffenderSummary[]> {
-    const caseAdminCaseload = await this.restClient.get<OffenderSummary[]>({
+    const caseAdminCaseload = await this.restClient.get<_OffenderSummary[]>({
       path: `/prison/${prisonCode}/case-admin/caseload`,
     })
     return caseAdminCaseload.map(c => {
-      return { ...c, hdced: format(c.hdced, 'dd MMM yyyy') }
+      return { ...c, hdced: parseIsoDate(c.hdced) }
     })
   }
```

Comment:
this receives `_OffenderSummary` from the API where `hdced` is a string.
It returns a `OffenderSummary` where `hdced` is a date.. this only compiles as we specifically parse the date on line 29.